io.github.am2r_community_developers.AM2RLauncher.yaml

```diff
@@ -0,0 +1,126 @@
+app-id: io.github.am2r_community_developers.AM2RLauncher
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+  - org.freedesktop.Sdk.Extension.openjdk11
+  - org.gnome.Sdk.Compat.i386
+
+finish-args:
+# TODO: crashes with wayland on filepicker and font being whacky. Likely Eto bug, test and report there.
+- --socket=x11
+- --share=ipc
+
+# Network needed for updates, pulseaudio for sound
+- --share=network
+- --socket=pulseaudio
+
+# Home for getting the 1.1 zip, config because of profiles having their saves there, apktool for apk creation
+- --filesystem=home:ro
+- --filesystem=~/.config/
```

Comment:
Your app should use `XDG_CONFIG_HOME` to save config files to.

Reply:
The programs the launcher launches are hardocded to use HOME/.config - they don't actually respect XDG_CONFIG_HOME. It's a limitation of the engine they use, which I can't get around.